api/src/main/java/uk/gov/dhsc/htbhf/claimant/reporting/ClaimAction.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.dhsc.htbhf.claimant.reporting;
+
+/**
+ * Action that triggered a claim event.
+ */
+public enum ClaimAction {
```

Comment:
Is there a reason for a new enum, or could we use `ClaimStatus` here?

Reply:
It's an action rather than a status, so for a new claim it's the same as the status, but other values will include transitions such as 'pending_expiry to expired' or 'active to expired'.